front/components/agent_builder/AgentBuilderFormContext.tsx

```diff
@@ -111,7 +108,40 @@ const searchActionSchema = baseActionSchema.extend({
 
 const dataVisualizationActionSchema = baseActionSchema.extend({
   type: z.literal("DATA_VISUALIZATION"),
-  configuration: dataVisualizationActionConfigurationSchema,
+  configuration: z.null(),
+});
+
+const reasoningModelConfigurationSchema = z.object({
+  providerId: z.string(),
+  modelId: z.string(),
+  temperature: z.number().nullable(),
+  reasoningEffort: z.string().nullable(),
+});
```

Comment:
Doesn't that conflict with / serve the same purpose as `generationSettingsSchema`? Let's try to unify ?